src/Uno.UI/ts/Windows/Storage/FileSavePicker.ts

```diff
@@ -0,0 +1,23 @@
+﻿namespace Windows.Storage.Pickers {
+
+	export class FileSavePicker {
+		public static SaveAs(fileName: string, dataPtr: any, size: number): void {
+
+			const buffer = new Uint8Array(size);
+
+			for (var i = 0; i < size; i++) {
+				buffer[i] = Module.getValue(dataPtr + i, "i8");
+			}
+
+			const a = window.document.createElement('a');
+			const blob = new Blob([buffer]);
+
+			a.href = window.URL.createObjectURL(blob);
+			a.download = fileName;
+
+			document.body.appendChild(a);
+			a.click();
```

Comment:
There's no more direct way of programmatically triggering a download?

Reply:
no... :)

Reply:
no... :)

Reply:
for something as common as a simple download, you would think there would be a more elegant way. But that's seems to be the way to do it.